### PR TITLE
[WholeGraph] Add missing step in WholeGraph Tutorial

### PR DESCRIPTION
**Pull request by isratnisa**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/advanced/advanced-wholegraph.rst [hunk 1]
```diff
@@ -22,15 +22,34 @@ Prerequisite
...
+    sudo systemctl restart docker
+
+Launch instance with EFA support
+--------------------------------------------------------
+
+While launching the EFA supported EC2 instances, in the Network settings section, choose Edit, and then do the following:
+
+    1. For Subnet, choose the subnet in which to launch the instance. If you do not select a subnet, you can't enable the instance for EFA.
+
+    2. For Firewall (security groups), choose Select existing security group, and then select the EFA-enabled security group
```
Comment (classicsong) [inline docs/source/advanced/advanced-wholegraph.rst:48]:
```suggestion
    2. For Firewall (security groups), choose Select existing security group, and then select the EFA-enabled security group or create one.
```
Can you also put a link to how to create one for EFA related security group?

Comment (isratnisa) [reply docs/source/advanced/advanced-wholegraph.rst:48]: The link is already in the per-requisite.

File: docs/source/advanced/advanced-wholegraph.rst [hunk 2]
```diff
@@ -22,15 +22,49 @@ Prerequisite
...
+
+    2. For Firewall (security groups), Choose `Select existing security group` and then pick the EFA-enabled security group you previously created as outlined in the prerequisites.
+
+    3. Expand the Advanced network configuration section, and for Elastic Fabric Adapter, select Enable.
+
+
+Install aws-efa-installer on the EC2 instance
+----------------------------------------------
+
+Install aws-efa-installer on both the base instance and within the Docker container. This enables the instance to use AWS's EFA kernel instead of Ubuntu's default kernel. Install the EFA installer without `--skip-kmod` on the instance and with --skip-kmod in the container. A command is provided for the base instance installation, and the Dockerfile will handle container installation in the next step.
```
Comment (classicsong) [inline docs/source/advanced/advanced-wholegraph.rst:56]: The following command is for `on the instance` or `in the container`?

**Review by classicsong — APPROVED** [review]

File: docs/source/advanced/advanced-wholegraph.rst (hunk 2 above)
Comment (isratnisa) [reply docs/source/advanced/advanced-wholegraph.rst:56]: on the instance.

**Pull request merged**